docs/usage.md

```diff
@@ -50,6 +50,26 @@ TREATMENT_REP3,AEG588A6_S6_L004_R1_001.fastq.gz,
 
 An [example samplesheet](../assets/samplesheet.csv) has been provided with the pipeline.
 
+## Filter/remove sequences from the samples (e.g. rRNA sequences with SILVA database)
+
+Metatdenovo can remove reads from the samples that matches specific sequences. By using BBMAP_BBduk module, it is possible to provide those sequences in fasta format.
+To add this function to the pipeline, just add the parameter `--sequence_filter sequence.fasta` to the command line or the `params.yml`file.
+You can check functionality and documentation of BBduk on the official [website](https://jgi.doe.gov/data-and-tools/software-tools/bbtools/bb-tools-user-guide/bbduk-guide/)
+
+## Digital normalization
+
+Metatdenovo can perform a digital normalization on the reads BEFORE the assembly. A digital normalization is useful for the assembly process because can reduce
+the number of over-rapresented sequences. N.B. the digital normalizatio is done only for the assembly while during the mapping process the reads will not be normalized.
+There are two to run the digital normalization:
```

Comment:
```suggestion
Metatdenovo can perform "digital normalization" on the reads BEFORE the assembly.
This will reduce coverage of highly abundant sequences and remove sequences that are below a threshold, and can be useful if the data set is too large to assemble but also potentially improve an assembly.
N.B. the digital normalization is done only for the assembly and the non-normalized sequences will be used for quantification.
There are two options for digital normalization in the pipeline:
```